### DISCUSSION TITLE
Shrine 3.5.0 released

### DISCUSSION THREAD

**Discussion by janko** [shrinerb/shrine#642] [Announcements]
After 2 long years since the last release, I'm happy to announce that Shrine 3.5.0 has been released. Some notable changes include:

* website has been migrated to Docusaurus v2 ✨ 
* support for custom signer in `derivation_endpoint` plugin
* `remove_attachment` plugin is Ruby 3.2 compatible

See full release notes here – https://shrinerb.com/docs/release_notes/3.5.0

**Comment by simonneutert** [comment]
Congratulations and big thanks to you @janko and everybody who helped! 🥳

**Comment by davidwessman** [comment]
Amazing! 🙂
